scripts/src/lib/ops/lint_contracts.ts

```
@@ -71,11 +72,21 @@ const APPROVED_OR_LATER = new Set([
   'completed',
 ]);
 
+// Sections that thin contracts omit — skip structural-completeness checks for these
+const THIN_SKIP_CHECKS = new Set([
```

Biome (2.5.7)

[error] This variable THIN_SKIP_CHECKS is unused.

(lint/correctness/noUnusedVariables)